screed/fadbm.py

```diff
@@ -17,3 +20,4 @@
     read_fasta_sequences(filename)
 
     print("Database saved in %s%s" % (sys.argv[1], DBConstants.fileExtension))
+    exit(0)
```

Comment:
For my education: What is the return code if you don't explicitly do this? Or are you being explicit because explicit is better than implicit?